types/react-svg-radar-chart/index.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for react-svg-radar-chart 1.2
+// Project: https://github.com/Spyna/react-svg-radar-chart
+// Definitions by: Lukas Tutkus <https://github.com/luksys5>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Typescript Version 3.5
+// Minimum TypeScript Version: 3.0
+
+import * as React from 'react';
+
+export interface ChartData {
+    data: {
+        [value: string]: number;
+    };
+    meta: { color: string };
+}
+
+export interface ChartOptionsProps {
+    size: number;
+    axes: boolean; // show axes?
```

Comment:
This inline comment does not work well with intelisense
```suggestion
    /** show axes */
    axes: boolean;
```

Reply:
Will remove it